Assets/Scripts/Storage/GlobalStorage_VM.cs

```diff
@@ -57,7 +57,7 @@ public static Chest_VM GetRandomChest()
             return null;
         }
 
-        int randomIndex = UnityEngine.Random.Range(0, chests.Count);
+        int randomIndex = UnityEngine.Random.Range(0, chests.Count - 1);
```

Comment:
Referring to the unity [documentation](https://docs.unity3d.com/ScriptReference/Random.Range.html), I believe when passing two integers to UnityEngine.Random.Range it's Range(minInclusive, maxExclusive).

Reply:
You're right. Completely missed that when I read the documentation. I completely overlooked that the section I was looking at was for `floats`, not `ints` as was used in `Chest_VM`. I sent a commit (fe56190) that reverts that change.